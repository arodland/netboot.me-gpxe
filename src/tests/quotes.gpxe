#!gpxe
#Tests for quoting. '
echo "ab"
#Spaces and escapes are permitted within " 
echo "Hello world,\ howdy?"
#Demonstrates escapes
echo Hello\ world. Don\'t shoot!
#Any character is permitted between ' and ', except another '
set a '"hello'
set b 'world"'
echo '${a} = '${a} '${b}'" = ${b}" # Show single/double quotes
echo Hello \
 World!
echo 'Hello
 World'
