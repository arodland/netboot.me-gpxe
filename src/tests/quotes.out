ab
Hello world, howdy?
Hello world. Don't shoot!
${a} = "hello ${b} = world"
Hello World!
Hello
 World
No more network devices

Press Ctrl-B for the gPXE command line...                                         
